arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java

```diff
@@ -257,8 +254,13 @@ public AggregationResponse formatAggregationResult(MultiBucketsAggregation aggre
                     if (subAggregationResponse.name.equals(FluidSearch.TERM_AGG)) {
                         subAggregationResponse.sumotherdoccounts = ((Terms) subAggregation).getSumOfOtherDocCounts();
                     }
-
-                    if (subAggregation.getName().equals(FluidSearch.DATEHISTOGRAM_AGG) || subAggregation.getName().startsWith(FluidSearch.GEOHASH_AGG) || subAggregation.getName().equals(FluidSearch.HISTOGRAM_AGG) || subAggregation.getName().equals(FluidSearch.TERM_AGG)) {
+                    if (subAggregation.getName().equals("fetched_hits")) {
```

Comment:
This seems like a bad use of map, you're adding something to a list from a forEach, can this not be replaced by a map?